lua/fishing_mod/catch/stove.lua

```diff
@@ -136,32 +135,48 @@ if CLIENT then
 			render.DrawSprite( self:GetPos() + ( self:GetUp() * v.position.z ) + ( self:GetRight() * v.position.x ) + ( self:GetForward() * v.position.y ), 1, 1, Color( 255, 255, 255, 255 ) )
 		end
 	end
-		
+
 	function ENT:ShowHeatAdjuster()
+		local back_ground = fishingmod.DefaultUIColors().ui_background
+		local button_not_selected = fishingmod.DefaultUIColors().ui_button_deselected
+		local ui_button_selected = fishingmod.DefaultUIColors().ui_button_selected
+		local ui_button_hovered = fishingmod.DefaultUIColors().ui_button_hovered
+		local ui_text = fishingmod.DefaultUIColors().ui_text
+		if fishingmod.ColorTable then
+			back_ground = fishingmod.ColorTable.ui_background or back_ground
+			button_not_selected = fishingmod.ColorTable.ui_button_deselected or button_not_selected
+			ui_button_selected = fishingmod.ColorTable.ui_button_selected or ui_button_selected
+			ui_button_hovered = fishingmod.ColorTable.ui_button_hovered or ui_button_hovered
+			ui_text = fishingmod.ColorTable.ui_text or ui_text
+		else
+			fishingmod.ColorTable = fishingmod.DefaultUIColors()
+		end
 		local frame = vgui.Create("DFrame")
 		frame.Paint = function(s, x, y)
-			surface.SetDrawColor(0, 0, 0, 144)
+			
+			surface.SetDrawColor(back_ground.r, back_ground.g, back_ground.b, back_ground.a)
 			surface.DrawRect(0, 0, x, y)
-			surface.DrawRect(3, 24, x-6, y-24-3)
+			surface.DrawRect(3, 24, x - 6, y - 24 - 3)

```

Comment:
if your -3 is a constant please make it a variable such as CONSTANT = 3, if not then just put the actual number instead of doing unnecessary operations